src/website/src/app/documentation/demos/spinners/spinner-component.html

```diff
@@ -0,0 +1,43 @@
+<!--
+  ~ Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h3 id="guidelines">Spinner Component</h3>
+<p>
+    Clarity provide component <code class="clr-code">clrSpinner</code>. 
+</p>
+
+<div class="clr-example squeeze">
```

Comment:
Have these behind an ngIf so they don't fire on load, and make a button to toggle loading for each demo.